src/db/collections/analytics/ledger_size.rs

```diff
@@ -25,15 +28,7 @@ pub struct LedgerSizeAnalyticsResult {
     pub total_storage_deposit_value: d128,
     pub total_key_bytes: d128,
     pub total_data_bytes: d128,
-}
-
-impl LedgerSizeAnalyticsResult {
-    pub fn total_byte_cost(&self, protocol_params: &ProtocolParameters) -> d128 {
-        let rent_structure = protocol_params.rent_structure;
-        d128::from(rent_structure.v_byte_cost)
-            * ((self.total_key_bytes * d128::from(rent_structure.v_byte_factor_key as u32))
-                + (self.total_data_bytes * d128::from(rent_structure.v_byte_factor_data as u32)))
-    }
+    pub total_byte_cost: d128,
```

Comment:
These should just use `u64`. The total byte cost can't exceed the number of tokens in the network. Same goes for the other fields.

Reply:
The problem is there is no way to convert `d128` to `u64` in the lib, and we have to use `Decimal128` in the query to ensure we have enough space. The only option I found is to convert through a string. Not sure it's worth it since we're just going to convert back to a string for influx.

Reply:
Ah yeah, makes sense.